src/content/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent.mdx

```diff
@@ -139,23 +123,25 @@ The preferred way to install the Linux agent is through the [package manager](/d
   />
 </TechTileGrid>
 
-To use these installation links, you must be logged to your New Relic account.
+If you don't have a New Relic account yet, or if you prefer to follow the procedure manually using our package manager, [see our tutorial](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-agent-linux-using-package-manager). 
 
-If you don't have a New Relic account yet, or if you prefer to follow the procedure manually, [see our tutorial](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-agent-linux-using-package-manager). For advanced install needs, you can deploy the agent using our tarball files in [assisted](/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux) or [manual](/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-manual-install-infrastructure-agent-linux) mode.
+For advanced install needs, you can deploy the agent using our tarball files in [assisted](/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux) or [manual](/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-manual-install-infrastructure-agent-linux) mode.
```

Comment:
I think we should remove this paragraph. We don't want to encourage customers to use the tarball installation.